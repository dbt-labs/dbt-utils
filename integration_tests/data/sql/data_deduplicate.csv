user_id,event,version
1,play,1
1,play,2
2,pause,1
