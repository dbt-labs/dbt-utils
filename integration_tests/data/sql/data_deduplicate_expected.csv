user_id,event,version
1,play,2
